### PR TITLE
14545 Upper case display for Corp Name Translation

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#14545

Corresponding PR in Edit UI https://github.com/bcgov/business-edit-ui/pull/433

*Description of changes:*
 - corp name translation input is now capitalized.
 - add unit test for name translation input being capitalized.
 - align AddNameTranslation.vue component and the unit tests to the Edit UI
 - App version 4.4.2 -> 4.4.3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/AddNameTranslation.vue [hunk 1]
```diff
@@ -9,25 +9,26 @@
         label="Enter Name Translation"
         v-model="nameTranslation"
         :rules="nameTranslationRules"
+        @input="nameTranslation = nameTranslation.toUpperCase()"
```
Comment (lambert-alex) [inline src/components/common/AddNameTranslation.vue:12]: The fix required by the ticket, all inputs are capitalized.

File: tests/unit/AddNameTranslation.spec.ts [hunk 2]
```diff
@@ -215,6 +223,18 @@ describe('Add Name Translation component', () => {
...
+
     wrapper.destroy()
   })
+
+  it('Inputed text is capitalized', async () => {
+    const wrapper = await wrapperFactory()
+    const translationInput = wrapper.find(addTranslationInput)
+    await translationInput.setValue('Lower case will be capitalized')
+
+    expect(wrapper.find(addTranslationInput).element.value).toBe('LOWER CASE WILL BE CAPITALIZED')
```
Comment (lambert-alex) [inline tests/unit/AddNameTranslation.spec.ts:238]: Unit test for the fix

File: src/components/common/AddNameTranslation.vue [hunk 3]
```diff
@@ -9,25 +9,26 @@
...
       />
 
       <div class="form__btns">
-        <v-btn large color="error" id="btn-remove"
+        <v-btn large color="error" id="name-translation-btn-remove"
```
Comment (lambert-alex) [inline src/components/common/AddNameTranslation.vue:16]: Aligning all of the btn IDs to be the same as the AddNameTranslation.vue in the Edit UI

Comment (severinbeauvais) [reply src/components/common/AddNameTranslation.vue:16]: Great, thanks. Did you also verify that the button ids are not used for styling? (ie, search for `#btn-remove`)

Comment (lambert-alex) [reply src/components/common/AddNameTranslation.vue:16]: Yes they were just used for testing, I also doubled checked the UI locally

**Comment by severinbeauvais** [comment]
I believe there is some "uppercase" CSS for the list of name translations that you can now delete.

**Review by ChrisGabel23 — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**